### PR TITLE
feat: configurable base branch name

### PR DESCRIPTION
**Pull request by TheAlbertDev**
Accepts a new input, `base_branch_name`, to indicate the name of
the branch containing the README file to be updated. Default
`main`.

### Why:

Closes https://github.com/skills/action-update-step/issues/4

### What's being changed:

```yml
  base_branch_name:
    description: "Option name of the base branch (default: main)."
    required: false
    default: "main"
```

### Check off the following:

- [x] For workflow changes, I have verified the Actions workflows function as expected.
- [ ] For content changes, I have reviewed the [style guide](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Review by sinsukehlab — APPROVED** [review]
Great!
This PR helps [CI course](https://github.com/skills/continuous-integration) to work.

**Comment by TheAlbertDev** [comment]
This repo doesn't seem very active... I have tagged the relevant commit in the fork (v1.2.0-beta) so that anyone who wants to can use the proposed changes until they are integrated into the repository someday. The only change to the GitHub action is:

```diff
- uses: skills/action-update-step@v1
+ uses: TheAlbertDev/action-update-step@v1.2.0-beta
```

**Comment by sinsukehlab** [comment]
Thanks for sharing.
I have edited yaml files in my generated CI course to use your fork and it works pretty well.
I think this repo is not very active because it is not a course.
Opening a relevant issue in CI course may make this repo more active.

**Review by Luis123QG — APPROVED** [review]
gh pr checkout 5

File: action.yml [hunk 1]
```diff
@@ -12,6 +12,10 @@ inputs:
     required: true
   branch_name:
     description: "If the learner is working in a branch, add the branch name."
+  base_branch_name:
+    description: "Option name of the base branch (default: main)."
```
Comment (heiskr) [inline action.yml:16]:
```suggestion
    description: "Optional name of the base branch (default: main)."
```

**Review by heiskr — APPROVED** [review]

**Comment by sinsukehlab** [comment]
`$BASE_BRANCH_NAME` env is not set.

**Review by Domsbrother — COMMENTED** [review]
Hi

**Pull request merged**